src/librustc_codegen_llvm/builder.rs

```diff
@@ -734,10 +830,10 @@ impl Builder<'a, 'll, 'tcx> {
         }
     }
 
-    pub fn inline_asm_call(&self, asm: *const c_char, cons: *const c_char,
-                           inputs: &[&'ll Value], output: &'ll Type,
-                           volatile: bool, alignstack: bool,
-                           dia: AsmDialect) -> Option<&'ll Value> {
+    fn inline_asm_call(&mut self, asm: *const c_char, cons: *const c_char,
+                       inputs: &[&'ll Value], output: &'ll Type,
+                       volatile: bool, alignstack: bool,
+                       dia: syntax::ast::AsmDialect) -> Option<&'ll Value> {
```

Comment:
Can this be moved to `asm.rs`? Actually, nothing in the trait interfaces should use raw pointers!